### PR TITLE
refactor redirects, routing, signin and signout

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#3258

*Description of changes:*
- added Dashboard URL to config
- updated Businesses URL in config
- redirect to dashboard on invalid NR error
- redirect to dashboard on account auth error
- don't render 'main' element during errors
- refactored initial fetch to work with signin
- deleted obsolete assets and components
- redirect to dashboard on cancel
- refactored Actions for error handling and consistent code
- return to dashboard after file & pay
- moved Keycloak init inside LD block
- redirect to Business Registy home page on startup error
- ensured Filing ID is a number
- refactored LegalApiMixin for error handling and consistent code
- refactored and fixed routing, signin and signout
- cleaned up config helper
- fixed unit tests, including added missing axios stubs
- misc other cleanup
- misc small fixes (see additional commits)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/mixins/filing-template-mixin.ts [hunk 1]
```diff
@@ -74,6 +76,12 @@ export default class FilingTemplateMixin extends Vue {
 
       // Set Persons and Organizations
       this.setOrgPersonList(draftFiling.incorporationApplication.parties)
+
+      // Set Certified By
+      this.setCertifyState({
+        certifyFormValid: false, // always override this
+        certifiedBy: draftFiling.header.certifiedBy
+      })
```
Comment (cameron-freshworks) [inline src/mixins/filing-template-mixin.ts:84]: nice 👍

**Comment by severinbeauvais** [comment]
> One thing I noticed probably a separate issue is when I go to 'Review and Confirm' and do a browser refresh the 'Incorporation' fee doubles up (2 entries).

I'll have a look. Don't merge yet in case it's an easy fix :)

**Comment by severinbeauvais** [comment]
> > One thing I noticed probably a separate issue is when I go to 'Review and Confirm' and do a browser refresh the 'Incorporation' fee doubles up (2 entries).
> 
> I'll have a look. Don't merge yet in case it's an easy fix :)

OK, I fixed this. It was due to a redirect when the Review and Confirm page was created. The data is now set asynchronously in App.vue so this redirect was incorrect and obsolete. Due to timing, fetchData() was called a second time before it completed the first time, and thus initialized the Fee Summary twice. It's only called once now!

**Comment by ochiu** [comment]
> > > One thing I noticed probably a separate issue is when I go to 'Review and Confirm' and do a browser refresh the 'Incorporation' fee doubles up (2 entries).
> > 
> > 
> > I'll have a look. Don't merge yet in case it's an easy fix :)
> 
> OK, I fixed this. It was due to a redirect when the Review and Confirm page was created. The data is now fetched asynchronously in App.vue so this redirect was incorrect and obsolete. Due to timing, fetchData() was called a second time before it completed the first time, and thus initialized the Fee Summary twice. It's only called once now!

Looks good! thanks!

**Pull request merged**